buildings_tools/src/buildings_tools/ToolSettings.java

```diff
@@ -21,12 +21,27 @@ private ToolSettings() {
     }
 
     public static final BooleanProperty PROP_USE_ADDR_NODE = new BooleanProperty("buildings_tools.addrNode", false);
+
+    public enum Shape {
+            CIRCLE, RECTANGLE
+    }
+
+    private static Shape shape = Shape.RECTANGLE;
     private static double width = 0;
     private static double lenstep = 0;
     private static boolean useAddr;
     private static final Map<String, String> TAGS = new HashMap<>();
     private static boolean autoSelect;
 
+    public static Shape getShape() {
+        loadShape();
```

Comment:
I don't think we want to read the pref every time the getter is called.  Wouldn't that make calling the setter (without calling save) rather ineffective?

Reply:
I'm starting to think we shouldn't even have a setShape() with the idea being we will always call saveShape() so they can pick up where they left off. I modeled this after getTags() and saveTags()